src/components/interactive-map.js

```diff
@@ -184,6 +184,14 @@ export default class InteractiveMap extends PureComponent {
     const controlOptions = Object.assign({}, this.props, {
       onChangeState: this._onInteractiveStateChange
     });
+    /*
+     * The map control needs `event.target` to calculate pointer positions
+     * relative to the map.
+     * If an overlay is interactive, when dragging over it, it becomes the
+     * `event.target`. Overwrite it with the event canvas here.
+     * The original target element is accessible in `event.srcEvent`.
+     */
+    event.target = this.refs.eventCanvas;
```

Comment:
> Overriding a field in the raw event feels odd to me. Happy to go with this for now, but maybe we should pass the eventCanvas down explicitly as well if that's what needed universally anyway?

Yes - it feels like the wrong place to do something so "low level", if we modify the event it should be done by EventManager as @Pessimistress suggested. Do we have all the information we need to do this in EventManager (access to the canvas etc)?